packages/core-database-sequelize/__tests__/repositories/transactions.test.js

```diff
@@ -170,6 +183,19 @@ describe('Transaction Repository', () => {
       })
     })
 
+    it('should find the same transactions with parameter senderId and corresponding senderPublicKey', async () => {
+      await connection.saveBlock(genesisBlock)
+      const senderWallet = await spv.walletManager.getWalletByPublicKey('034776bd6080a504b0f84f8d66b16af292dc253aa5f4be8b807746a82aa383bd3c')
+
+      const transactionsSenderPublicKey = await repository.findAllLegacy({ senderPublicKey: senderWallet.publicKey })
+      const transactionsSenderId = await repository.findAllLegacy({ senderId: senderWallet.address })
+
+      transactionsSenderPublicKey.rows.forEach(transactionSenderPublicKey => {
+        const transactionSenderId = transactionsSenderId.rows.find(tr => tr.id === transactionSenderPublicKey.id)
```

Comment:
This wouldn't test that the number of transactions is the same, their order, etc.